packages/fxa-payments-server/src/components/fields.tsx

```diff
@@ -75,6 +77,14 @@ export const Field = ({
   );
   /* eslint-enable react-hooks/exhaustive-deps */
 
+  useEffect(() => {
+    if (autoFocus && tooltipParentRef && tooltipParentRef.current) {
+      setTimeout(() => {
```

Comment:
So, I'm actually kind of unsure about this PR: This seems to intermittently not work on the payment page. 😞 Might need a second opinion on that page - I think something else on there is stealing focus, but I can't tell what. This `setTimeout` is a brute force attempt to steal back focus after a vague wait.

Reply:
Yeah, this is odd, the `body` doesn't want to lose focus. The `useLayoutEffect` hook has no affect and the examples I've found online suggest to do it the way you are here. I'll see if I can keep digging since this does feel pretty hacky😅

Reply:
Yeah, I don't like what I've done here at all. 😓

Reply:
Well, I tried conditionally rendering `autoFocus` based on `childProps` in `input` rather than fields, then playing with the hook and hook dependencies, and looking up a ton of stuff on SO with no dice... I've set `setTimeOut` to `0` and thrown a bunch of logs in product, payment form, and fields, to see if I could catch something weird with render timing, and even when `setTimeout` logs at the very end, `0` consistently does not work so it's very difficult to identify what is causing the `body` to regain focus.

Have we not had React dev tools working the entire time on payments? It tells me it's not a React application. That could come in handy for debugging something like this for highlighting rerenders. 🤔 I also can't connect to dev tools React on staging:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/13018240/67820470-61367a80-fa87-11e9-9e66-adb9daa1c679.png">

If we can get React dev tools working to help see what's happening (maybe I'm missing something on getting it working?) this might be worth revisiting before merging... Otherwise, we might want to ship it with a "TO DO" comment written above. 😩